src/darker/tests/test_git.py

```diff
@@ -449,3 +449,19 @@ def test_edited_linenums_differ_revision_vs_lines(git_repo, context_lines, expec
     result = differ.revision_vs_lines(Path("a.py"), content, context_lines)
 
     assert result == expect
+
+
+def test_local_gitconfig_ignored_by_gitrepofixture(tmp_path):
+    """Tests that ~/.gitconfig is ignored when running darker's git tests"""
+    with open(tmp_path / "HEAD", "w") as head_file:
+        head_file.write("ref: refs/heads/main")
```

Comment:
This could be written as
```suggestion
    (tmp_path / "HEAD").write_text("ref: refs/heads/main")
```

Reply:
TIL. old habits die hard.